src/commonMain/kotlin/exchange.dydx.abacus/processor/markets/MarketProcessor.kt

```diff
@@ -111,6 +112,8 @@ internal class MarketProcessor(parser: ParserProtocol, private val calculateSpar
         "double" to mapOf(
             "volume24H" to "volume24H",
             "openInterest" to "openInterest",
+            "openInterestLowerCap" to "openInterestLowerCap",
+            "openInterestUpperCap" to "openInterestUpperCap",
```

Comment:
honestly have no idea what this is for. don't even know if i need this

Reply:
ah yeah i see now, in `private fun perpetual` it's used as an input to the `transform` method. makes sense